README.md

```diff
@@ -3,43 +3,47 @@
 Bridge selected events from AppStudio into [Segment][1]
 
 ```mermaid
-flowchart TD
+flowchart TB
     subgraph A["RHTAP clusters"]
         A1[(API server)]
-        A2(["OpenShift
-         logging collector"])
+        A2(["`OpenShift
+        logging collector`"])
 
         A1--"Audit Logs"-->A2
     end
 
-    A2--"Audit Logs"--->C1
+    A2--"Audit Logs"-->C
     
-    subgraph C["Splunk"]
-        C1[("Splunk index")]
-        C2[("Splunk KV store")]
-        C3(["Splunk search"])
-
-        C1 & C2-->C3
-    end
+    C[("Splunk")]
 
     A1--"UserSignup resources"-->B1
 
-    B1--"Username to SSO ID mapping"-->C2
-
     subgraph B["RHTAP Segment bridge"]
-        B1([User mapper])
-        B2([Event sender])
-
-        B1-..->B2
+        B1([get-uid-map.sh])
+        B2([fetch-uj-records.sh])
+        B3([splunk-to-segment.sh])
+        subgraph B4[segment-mass-uploader.sh]
+            B4C([split])
+            B4A([segment-uploader.sh])
+            B4B([mk-segment-batch-payload.sh])
+
+            B4C--"Segment events (In ~490KB batches)"-->B4A
+            B4A--"events"-->B4B--"batch call payload"-->B4A
+        end
+
+        B1--"Username to UID map"-->B3
+        B2--"Splunk-formatted UJ records"-->B3
+        B3--"Segment events"--> B4
     end
 
-    C3--"User resource
-     actions"-->B2
+    C-- "User resource
+     actions" -->B2
 
     G([Segment])
     H[(Aplitude)]
```

Comment:
is that a typo? Can we fix it while we're at it? 
```suggestion
    H[(Amplitude)]
```

Reply:
fixed.